vmdb/app/controllers/ems_common.rb

```diff
@@ -601,6 +608,14 @@ def set_form_vars
     @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(EmsAmazon)
     @edit[:new][:port] = @ems.port
     @edit[:new][:provider_id] = @ems.provider_id
+    @edit[:protocols] = [['Basic (SSL)', 'ssl'], ['Kerberos', 'kerberos']]
```

Comment:
Looks like this belongs in a translation file...
Maybe at least include a TODO comment here.